src/strategies/music-os-accumulator/extractor.mjs

```diff
@@ -179,6 +196,7 @@ export async function init() {
   trackList = [...trackList, ...strategies[3].map];
   trackList = [...trackList, ...strategies[4].map];
   trackList = [...trackList, ...strategies[5].map];
+  trackList = [...trackList, ...Array.from(strategies[6].map.values())];
```

Comment:
For each value of `strategies[6].map.values()` we must only take e.g. the first value.